components/raftstore/src/coprocessor/region_info_accessor.rs

```diff
@@ -97,8 +104,14 @@ impl RegionInfo {
     }
 }
 
+#[derive(Clone, Debug)]
+pub struct RegionActivity {
+    pub region_stat: RegionStat,
```

Comment:
maybe we can add a TODO here to record the region's MVCC version/tombstone count which can be used to measure the effectiveness of the in-memory cache for that region's data.  These information can be collected from rocksdb, example is collect_regions_to_compact.

Reply:
done.